addons/material-forum-editor-buttons/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Material forum editor buttons",
+  "description": "Changes the forum editor icons from tiny 16x16 bitmaps to SVG Google Material Icons.",
+  "userstyles": [
+    {
+      "url": "icons.css",
+      "matches": ["https://scratch.mit.edu/discuss/*"]
+    }
+  ],
+  "tags": ["community", "forums", "theme", "beta", "recommended"],
```

Comment:
Not all of these should be here

Reply:
remove reccomended - that's w_l's call, not yours
remove beta - this doesn't need to be beta, it's small enough, what wouldn't work?
remove community - forums automatically implies community

Reply:
> forums automatically implies community

wrong

Reply:
oh I thought it did since the prettier code blocks addon only had forums not community

Reply:
> oh I thought it did since the prettier code blocks addon only had forums not community

I shouldn't be any example, people pay less attention to my PRs.